packages/mesos/build

```diff
@@ -17,8 +17,7 @@ LIBS="-lssl -lcrypto" LDFLAGS="-L/opt/mesosphere/active/openssl/lib" "/pkg/src/m
   --with-libevent=/opt/mesosphere/active/libevent \
   --with-curl=/opt/mesosphere/active/curl \
   --sbindir="$PKG_PATH/bin"
-# TODO(cmaloney): -j8 is almost certainly wrong.
-make -j8
```

Comment:
Two comments.

1. `-j` without any args is supposed to utilize all the cpus.  It can be verified.
2. Utilizing all cpus is not error-free. only by benchmarking can we arrive a good value for `j`